plugins/chat/app/services/chat/publisher.rb

```diff
@@ -35,7 +37,21 @@ def self.publish_new!(chat_channel, chat_message, staged_id)
       )
     end
 
+    def self.publish_thread_created!(chat_channel, chat_message)
+      content =
+        Chat::MessageSerializer.new(
+          chat_message,
+          { scope: anonymous_guardian, root: :chat_message },
+        ).as_json
+      content[:type] = :thread_created
+      permissions = permissions(chat_channel)
+
+      MessageBus.publish(root_message_bus_channel(chat_channel.id), content.as_json, permissions)
+    end
+
     def self.publish_processed!(chat_message)
+      return if chat_message.thread_id.present? && !chat_message.is_thread_om?
```

Comment:
I think it would be worth extracting this logic `chat_message.thread_id.present?` into some `Chat::Message` model instance method, eg: `message.in_thread?` or whatever you think would be better in english

Reply:
In this case we could even go further and also have a `message.thread_reply?` which would be the equivalent of `chat_message. in_thread? && !chat_message.is_thread_om?`

Reply:
Yes good idea, will do that 👍 This publisher will get more refactors in the next PR too when I hook up the thread events properly